lib/core/typeinfo.nim

```diff
@@ -67,11 +67,11 @@ type
   pbyteArray = ptr array[0.. 0xffff, int8]
 
   TGenericSeq {.importc.} = object
-    len, space: int
+    len, space, elemSize: int
   PGenSeq = ptr TGenericSeq
 
 const
-  GenericSeqSize = (2 * sizeof(int))
+  GenericSeqSize = (3 * sizeof(int))
```

Comment:
I need the elemSize field in TGenericSeq for growObj().

Since the Go GC doesn't have a realloc(), I need to know the sequence's size and the trick done by the default GC (putting a TCell at the start of the allocated region) can't be used. The Go GC scans for pointers to the beginning of the allocated region, not offseted pointers.

So since I need to increase GenericSeqSize in lib/system.nim I might as well increase it in lib/core/typeinfo.nim.